### ISSUE TITLE
`with autocast("cuda")` causes cuda memory bugs on inductor

### ISSUE THREAD

**Issue by voznesenskym** [pytorch/torchdynamo#1101]
This is `from torch import autocast`. Tested on a single model so far. Works with aot_nvfuser.

**Comment by anijain2305** [comment]
Do you have a specific repro? We have many tests running with Inductor with autocast in nightly. So, it might be related to some specific kernel.

**Comment by voznesenskym** [comment]
Yeah, I'll add repro errors, it could def be a specific kernel...
